src/lib/components/Identifiers/PIDField.js

```diff
@@ -22,6 +22,13 @@ import { DISCARD_PID_STARTED, RESERVE_PID_STARTED } from '../../state/types';
 const PROVIDER_EXTERNAL = 'external';
 const UPDATE_PID_DEBOUNCE_MS = 200;
 
+const getFieldErrors = (form, fieldPath) => {
+  return (
+    getIn(form.errors, fieldPath, null) ||
+    getIn(form.initialErrors, fieldPath, null)
```

Comment:
I think you can remove the label also from here https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/components/FormFeedback.js#L51